test/app-luatest/uuid_test.lua

```diff
@@ -0,0 +1,172 @@
+local uuid = require('uuid')
+local t = require('luatest')
+
+local g = t.group()
+
+g.test_rfc4122 = function()
+    local u = uuid.new()
+    t.assert_equals(bit.band(u.clock_seq_hi_and_reserved, 0xc0), 0x80,
+                    '<uuid.new> always generates RFC4122 variant')
+    local version = bit.rshift(u.time_hi_and_version, 12)
+    t.assert(version == 1 or version == 4,
+             '<uuid.new> generates time-based or random-based version')
```

Comment:
It is always v4, isn't it?

Reply:
I suppose this is needed for FreeBSD, where [the `uuidgen()` syscall](https://man.freebsd.org/cgi/man.cgi?query=uuidgen&sektion=2&n=1) is used for the uuid generation. It always generates the v1 uuid, IINM. I will add the corresponding comment.

Reply:
Added the comment.

Reply:
This is a surprising behavior: I wouldn't expect that it is platform dependent. Anyway, it is out of scope of this pull request. I've filed a documentation issue: https://github.com/tarantool/doc/issues/4333.